enarx-keep-sgx/src/builder.rs

```diff
@@ -137,8 +137,8 @@ impl Builder {
                 }
             }
 
-            let line = Line::from(span);
-            eprintln!("{:016x}-{:016x} {:?}", line.start, line.end, si);
+            //let line = span::Line::from(span);
+            //eprintln!("{:016x}-{:016x} {:?}", line.start, line.end, si);
```

Comment:
Should we just delete these lines?

Reply:
No. These are incredibly useful for debugging. But I agree we probably need a better strategy. This is something that we need to sync sgx/sev on. Do we want to enable debug output with debug builds and disable for release builds? @connorkuehl @haraldh ?

Reply:
My experience with customers using my software taught me, that it is incredibly useful to have some debugging switches, which show lots of debugging useful for bug reports.

Reply:
Yes, I'd love debug build output.

Reply:
Normally I do:
```rust
#[cfg(debug_assertions)]
    eprintln!(...)
```
which prints for non `--release` builds... as long as `-C opt-level=3` is not specified in `.cargo/config` :cry:

Reply:
> which prints for non --release builds... as long as -C opt-level=3 is not specified in .cargo/config :cry:

That's probably ok, I can't imagine a debug build with opt-level 3 :nauseated_face: 

Though I'd prefer something like

```rust
#[cfg(debug)]
eprintln!("blorp");
```

But that syntactic sugar would require us to put some `cfg_attr` in a common area for each crate.

```rust
#[cfg_attr(debug_assertions, debug)]
```

Reply:
@connorkuehl @haraldh We have an added problem though. We can't allow the host to turn on debugging switches since this leaks an incredible amount of information to the host. This is why I was thinking of gating this on the build release profile. For example, if you're running on Enarx built with `cargo build --release` you get no debugging messages and we don't enable the debug flag for the underlying technology. However, on the non-release builds, the debug flag for the underlying technology is set to on and the keep will dump debug output.

Reply:
@npmccallum @haraldh Would we have the same problem if a host built their own debug packages of Enarx?

Reply:
The keep measurement would differ from the official release. So simply by attesting you know it isn't in debug mode.